extensions/wikia/CommunityPage/CommunityPageSpecialController.class.php

```diff
@@ -71,6 +71,13 @@ public function getTopContributorsData() {
 
 		$topContributorsDetails = $this->getContributorsDetails( $topContributors );
 
+		global $wgLang;
+		$redirect = 'redirect=' . urlencode( SpecialPage::getTitleFor( 'CommunityPage' )->getCanonicalURL() );
+		$uselang  = 'uselang=' . $wgLang->getCode();
+		$login = '<a href="https://www.wikia.com/signin?' . $redirect . '&' . $uselang . '">' . $this->msg( 'communitypage-anon-login' )->escaped() . '</a>';
+		$register = '<a href="https://www.wikia.com/register?' . $redirect . '&' . $uselang . '">' . $this->msg( 'communitypage-anon-register' )->escaped() . '</a>';
+		$anonText = $this->msg( 'communitypage-anon-contrib-header', $login, $register )->plain();
```

Comment:
The message should still be escaped, so if you need to pass raw HTML variables like this, use `rawParams`, i.e. `$this->msg( 'communitypage-anon-contrib-header' )->rawParams( $login, $register )->escaped();` This injects the parameters unescaped after the main message is safely escaped, but should of course only be used if all the raw parameters are known to be safe.